### DISCUSSION TITLE
How to add the symbol of fixed phone?

### DISCUSSION THREAD

**Discussion by ifantasy** [posquit0/Awesome-CV#602] [Q&A]
When I tried to add the command _\phone[fixed]{}_, it didn't work. So, how to add the symbol of fixed phone?
And, if I would like to add two email addresses, how to use the command _\email{}_? That two email addresses share one symbol is the best choice.

**Comment by johannesbottcher** [comment]
Last question is simple, very simple. `\email{posquit0.bj@gmail.com, user@userbase.com}`.

First question: The template does not provide a macro for `phone`.
